Malaria App Widget/TodayViewController.swift

```diff
@@ -23,7 +23,8 @@ class TodayViewController: UIViewController, NCWidgetProviding {
   }
 
   // Solves unnecesary widget bottom margin problem
-  func widgetMarginInsetsForProposedMarginInsets(var defaultMarginInsets: UIEdgeInsets) -> UIEdgeInsets {
+  func widgetMarginInsetsForProposedMarginInsets(defaultMarginInsets: UIEdgeInsets) -> UIEdgeInsets {
+    var defaultMarginInsets = defaultMarginInsets
```

Comment:
That line solves attributing a value to a variable function parameter (on the next line), making the code compatible with Swift 3, where "var paramaters" are not allowed anymore.

I followed this suggestion and I am open to any other alternatives.

https://github.com/apple/swift-evolution/blob/master/proposals/0003-remove-var-parameters.md